### PR TITLE
[SWARM-584] Upgrade to wildfly-camel-4.2.0

### PR DESCRIPTION
**Pull request by tdiesler**

### CODE REVIEW

**Comment by tdiesler** [comment]
ok to test

File: camel/pom.xml [hunk 1]
```diff
@@ -40,7 +40,7 @@
     <!-- Properties -->
     <properties>
         <version.apache.camel>2.17.1</version.apache.camel>
```
Comment (jamesnetherton) [inline camel/pom.xml]: This needs bumping to 2.17.2, right?

Comment (tdiesler) [inline camel/pom.xml]: yes, merci

**Comment by kenfinnigan** [comment]
ok to test

**Pull request merged**